src/main/java/sirius/biz/scripting/mongo/MongoCustomEventDispatcherRepository.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.scripting.mongo;
+
+import sirius.biz.scripting.CustomEventDispatcher;
+import sirius.biz.scripting.CustomEventDispatcherRepository;
+import sirius.biz.scripting.CustomEventRegistry;
+import sirius.biz.scripting.ScriptBasedCustomEventDispatcher;
+import sirius.db.mongo.Mango;
+import sirius.kernel.async.TaskContext;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.health.HandledException;
+import sirius.kernel.tokenizer.Position;
+import sirius.pasta.noodle.Callable;
+import sirius.pasta.noodle.ScriptingException;
+import sirius.pasta.noodle.SimpleEnvironment;
+import sirius.pasta.noodle.compiler.CompilationContext;
+import sirius.pasta.noodle.compiler.NoodleCompiler;
+import sirius.pasta.noodle.compiler.SourceCodeInfo;
+import sirius.pasta.noodle.sandbox.SandboxMode;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Stores and manages {@link CustomEventDispatcher custom event dispatchers} in a MongoDB.
+ */
+@Register(framework = MongoCustomEventDispatcherRepository.FRAMEWORK_SCRIPTING_MONGO)
+public class MongoCustomEventDispatcherRepository implements CustomEventDispatcherRepository {
+
+    protected static final String FRAMEWORK_SCRIPTING_MONGO = "biz.scripting-mongo";
```

Comment:
**Question:** Is this a new framework? JavaDoc may be nice in that case.

Reply:
done